site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -316,7 +316,13 @@ If you have out of scope questions and have not purchased a support contract, yo
 
 If managed FusionAuth hosting does not meet your needs, open a link:/contact/[contact us] with more details.
 
-For instance, if you need longer retention of database backups or static IP addresses capable of being added to a firewall allow/deny list, we're happy to discuss that with you.
+We're happy to discuss custom deployment architectures or configurations. If you need any of these:
+
+* Longer retention of database backups.
+* Static IP addresses capable of being added to a firewall allow/deny list.
```

Comment:
Do we really support this?

Reply:
We could if someone asked and were willing to pay. Elastic ip address, or run everything through a proxy with a fixed ip address.